SalesforceCommerceCloud/pwa-kit
# Increase test timeouts only on CI env

This PR gives some breathing room to the CI tests, so that any document queries or other things that may run intermittently slower can still finish without running into timeouts (timeouts from either react-testing-library or Jest).

In your local machine though, the tests should run much faster and thus not needing the increased timeouts there.

## How to test the changes
To verify that _local_ tests still fail fast with the default timeouts, try updating one of the Checkout tests like this:

```js
// Inside the file app/pages/checkout/index.test.js
test.only('Renders skeleton until customer and basket are loaded', async () => {
    const {getByTestId, queryByTestId} = renderWithProviders(<Checkout />)

    expect(getByTestId('sf-checkout-skeleton')).toBeInTheDocument()
    expect(queryByTestId('sf-checkout-container')).not.toBeInTheDocument()

    console.time('FOO')
    try {
        await waitFor(() => {
            expect(getByTestId('FOO')).toBeInTheDocument()
        })
    } catch {
        // The above query should fail because FOO test id does not exist
    }
    console.timeEnd('FOO')
})
```

Then run the test by `npm run test -- "./app/pages/checkout/index.test.js --no-coverage"`. 
The FOO timer should last for around the default 1 sec (and not 6 seconds).

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- (step1)

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
